mitodl/mit-open
# follow/unfollow popover

### What are the relevant tickets?

Closes https://github.com/mitodl/hq/issues/5385

### Description (What does it do?)
This PR introduces a follow action popover when clicking 'follow' on channel pages as described in https://github.com/mitodl/hq/issues/5385

### Screenshots (if appropriate):
<img width="363" alt="Screenshot 2024-09-23 at 11 34 08 AM" src="https://github.com/user-attachments/assets/95dc7f26-e377-4766-922e-9a16fb731e9c">
<img width="353" alt="Screenshot 2024-09-23 at 11 34 20 AM" src="https://github.com/user-attachments/assets/52a638f3-50b7-4915-aff7-2c6bcec58b82">


### How can this be tested?
1. checkout this branch (or start a codespace)
2. visit a unit, department or topic channel page and click follow
3. as an anonymous user you should see the signup action popover - logged in users will see a follow/unfollow or close action popover
